src/universal/styles/ui.js

```diff
@@ -483,7 +480,7 @@ const ui = {
   // Menus
   // ---------------------------------------------------------------------------
   menuBackgroundColor: '#fff',
-  menuBorderColor: appTheme.palette.mid30l,
+  menuBorderColor: '#DFDFE5',
```

Comment:
looks like this is only used once. can we remove it from ui.js and replace with BORDER.LIGHTER?